projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php

```diff
@@ -1435,7 +1485,13 @@ public static function get_related_accounts( $post ) {
 	 * Add content specific to a service in the footer.
 	 */
 	public function display_footer() {
-		$this->js_dialog( $this->shortname, array( 'height' => 350 ) );
+		if ( $this->smart ) {
+			?>
+			<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
```

Comment:
I have not updated the domain name here since I still get redirected to `platform.twitter.com` in my tests.